bot/src/discord/commands/DynamicSoundCommand.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  ActionRowBuilder,
+  ApplicationCommandOptionType,
+  ButtonBuilder,
+  ButtonStyle,
+  ChatInputCommandInteraction,
+  MessageActionRowComponentBuilder,
+  PermissionFlagsBits,
+} from 'discord.js';
+import { prismaClient } from '../../bot';
```

Comment:
Guessing the flag you're passing to node means you don't need to append the `.js` to the import flag here?

Reply:
Yes, see the other response about using esm.